landing_page/templates/landing_page/movie.html

```diff
@@ -1 +1,12 @@
-{% extends 'landing_page/base.html' %} {% load static %} {% block content %}
\ No newline at end of file
+{% extends 'landing_page/base.html' %} {% load static %} {% block content %}
+{% if movies %}
+        <div>Movies found</div>
+        <ul>
+            {% for movie in movies %}
+                <li>{{ movie }}</li>
+            {% endfor %}
+        </ul>
+    {% else %}
+        <div>No movies passed</div>
+    {% endif %}
+{% endblock %}
```

Comment:
Just to note, Elizabeth should have some code coming into this file soon but its for movie details and not searching which I believe this is for? I can let her know to move the html into movie_details.html which I think she was unaware of.

Reply:
Right now it can be removed because that was just for testing